nucypher/characters/lawful.py

```diff
@@ -462,12 +462,14 @@ class IncorrectCFragsReceived(Exception):
         def __init__(self, evidence: List):
             self.evidence = evidence
 
-    def __init__(self, controller: bool = True, *args, **kwargs) -> None:
+    def __init__(self, treasure_maps: Dict = dict(), controller: bool = True, *args, **kwargs) -> None:
```

Comment:
Don't use a mutable default argument -- this may have nasty unintentional side-effects when it's called more than once.  Instead, use `None` and initialize the empty dict inside `__init__.py`

Reply:
:+1: